tests/framework/cli/test_jupyter.py

```diff
@@ -141,7 +141,12 @@ def test_create_new_kernel(self):
         assert kernel_spec.language == "python"
         assert kernel_spec.argv[-2:] == ["--ext", "kedro.ipython"]
         kernel_files = {file.name for file in Path(kernel_spec.resource_dir).iterdir()}
-        assert kernel_files == {"logo-32x32.png", "logo-64x64.png", "kernel.json"}
+        assert kernel_files == {
+            "logo-32x32.png",
+            "logo-64x64.png",
+            "kernel.json",
+            "logo-svg.svg",
```

Comment:
```suggestion
            "kernel.json",
            "logo-32x32.png",
            "logo-64x64.png",
            "logo-svg.svg",
```
Nit: Absolutely unnecessary, but I just felt the order was arbitrary, so this is alphabetized and matching the docstring for `_create_kernel`. 😂